source/grammar/tests/outputs/sub_and_kern_call.yaml

```diff
@@ -1,6 +1,6 @@
 # indent w/ 2 spaces
 source: |
-  bit x = sub_call(10, "01") q1[0], q2;
+  bit x = sub_call(10, "01", q1[0], q2);
```

Comment:
Can you add index identifier specifics such as either `||` or `[:]` syntax to the test?

Reply:
I'm not sure that such operators are allowed excepting alias statements.